pdart/pds4/HstFilename.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, filename: str) -> None:
         ), "Filename must be at least six characters long"
         basename2 = basename(filename)
         assert (
-            basename2[0].lower() in "ijnu"
+            basename2[0].lower() in "ijlnouwxyz"
```

Comment:
Change to `basename2[0].upper() in ACCEPTED_INSTRUMENTS` so we don't have multiple copies of the same string that can get out of sync with each other.  My mistake in not doing that earlier.